akd/src/helper_structs.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
nit: Given the level of use these structs get, can we re-export ones like ```Node<H>``` so that it appears publicly as ```akd::Node```? Having a different module for these structs is good for code organization, but they're quite common and public use will just get messy I think.